include/sys/byteorder.h

```diff
@@ -139,6 +139,21 @@ static inline void sys_put_be16(u16_t val, u8_t dst[2])
 	dst[1] = val;
 }
 
+/**
+ *  @brief Put a 24-bit integer as big-endian to arbitrary location.
```

Comment:
Perhaps mention that the 24-bit integer is right justified?

Reply:
I did not quite get by right justification. You mean, the source is right justified in the CPU endian u32_t that gets put into 3-byte destination?

Reply:
A 24-bit quantity in a 32-bit register could either be right justified (0x00FF_FFFF) or left-justified (0xFFFF_FF00). Left justification is sometimes used for fixed-point fractional numbers

Reply:
Agreed. Will update the comments.